actions/conversation-v1/create-counterexample.js

```diff
@@ -64,13 +64,18 @@ module.exports.test = main;
 function getParams(theParams, theServiceName) {
   if (Object.getOwnPropertyNames(theParams).length === 0) {
```

Comment:
```
(Object(theParams).keys().length === 0)
```

Reply:
this results in an error for me -- i think it should be:  `Object.keys(theParams).length === 0`